server/model/Game.go

```diff
@@ -27,23 +27,37 @@ type Game struct {
 
 // GameSummary Provides summary information for the lobby
 type GameSummary struct {
-	ID      string     `json:"id"`
-	Name    string     `json:"name"`
-	Creator string     `json:"creator"`
-	Players []string   `json:"players"`
+	ID      string `json:"id"`
+	Name    string `json:"name"`
+	Creator string `json:"creator"`
+	// Players []string `json:"players"`
+	Players []Player   `json:"players"`
 	Status  GameStatus `json:"status"`
 }
 
+// GameToSummary Converts a Game to a GameSummary
```

Comment:
Do we still need this if it is commented out?

Reply:
I just don't know if we'll need it later. If the summary is used somewhere and they use it `game.players[idx]` then before it would give you the name of the idx, but nowyou'd need to use `game.players[idx].name`

Reply:
I didn't see any errors though do you think that's good enough?